tracee-ebpf/tracee/tracee.go

```diff
@@ -713,6 +713,18 @@ func (t *Tracee) populateBPFMaps() error {
 	stringStoreMap, _ := t.bpfModule.GetMap("string_store")
 	stringStoreMap.Update(uint32(0), []byte("/dev/null"))
 
+	kernelConfig := helpers.KernelConfig{}
+	kconf, err := helpers.InitKernelConfig(kernelConfig)
+	if err != nil {
+		return fmt.Errorf("could not read in kernel configuration %v", err)
```

Comment:
Do we really want to fail tracee in this case? Maybe just report to the user and continue - things may still work even if we fail to read the config

Reply:
Good point